_posts/2024-06-01-opensearchcon-north-america-2024-registration-and-cfp.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title:  "Announcing OpenSearchCon North America 2024"
+authors:
+  - daryllswager
+date:   2024-06-01 0:00:10 -0700
+categories:
+  - community
+excerpt: The OpenSearch Project invites the OpenSearch community to explore the future of search, analytics, and generative AI at the first OpenSearch user conference in Europe. Join us in Berlin on May 6 & 7 and learn how to build powerful applications and get the most out of your OpenSearch deployments.
+meta_keywords: opensearchcon Europe, opensearchcon eu, opensearchcon emea, opensearchcon may, opensearchcon call for papers, opensearchcon call for presentations, opensearchcon registration, register for opensearchcon
+meta_description: OpenSearchCon Europe 2024 is coming to Café Moskau in Berlin May 6 and 7. Register today to save your seat!
```

Comment:
Updated meta: 
**_meta_keywords:_**  opensearchcon north america, opensearchcon na, opesearchcon call for papers, register for opensearchcon, opensearch community

**_meta_description:_** Join the OpenSearch Project in San Francisco for it’s third annual OpenSearchCon North America 2024 taking place September 24-26 at the Hilton Union Square. Register today.

Reply:
All set thanks Patti.

Reply:
I'm assuming this text should reference the North American conference.